exe/solr_wrapper

```diff
@@ -73,6 +93,18 @@ end
 options[:verbose] = true if options[:verbose].nil?
 
 instance = SolrWrapper.instance(options)
+
+if command == 'clean'
+  if instance.started?
+    $stderr.print "Please stop soll before cleaning\n"
+    exit 1
+  end
+  $stderr.print "cleaning #{instance.instance_dir}...\n"
+  instance.remove_instance_dir!
+  instance.extract_and_configure
```

Comment:
Yeah, I don't see any reason to immediately extract solr again, so we can just be lazy.